test/unit/spec/MfaVerify_spec.js

```diff
@@ -274,12 +274,12 @@ Expect.describe('MFA Verify', function () {
     const res = JSON.parse(JSON.stringify(resMfaAlwaysPolicy));
 
     if (options) {
-      if (options.hasOwnProperty('minutes')) {
+      if (Object.prototype.hasOwnProperty.call(options, 'minutes')) {
```

Comment:
Just feels like `options.minutes` work here right? Do we really need to do `hasOwnProperty`?

Reply:
Done